internal/blueprint/customizations_test.go

```diff
@@ -0,0 +1,158 @@
+package blueprint
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestCustomizationsHostname(t *testing.T) {
+
+	var myHostname = "MyHostname"
+
+	TestCustomizations := Customizations{
+		Hostname: &myHostname,
+	}
+
+	bpOrig := Blueprint{
+		Name:           "customizations-test",
+		Description:    "Testing customizations functions",
+		Version:        "0.0.1",
+		Customizations: &TestCustomizations,
+	}
+
+	assert.Equalf(t, &myHostname, TestCustomizations.GetHostname(), "The hostname is different")
+	assert.Equalf(t, &myHostname, bpOrig.Customizations.GetHostname(), "The hostname is different")
```

Comment:
Question: why do we have 2 asserts which call GetHostName via 2 different paths? I don't understand the scenario here.

Reply:
That is actually related to my 3rd question in my first comment. I can access the `Customizations struct` on it's own or put in inside a `Blueprint struct` and call the method from there. My question was if there's any practical difference or if that makes sense to check it both ways, but @ondrejbudai has already answered me that it should not bring any benefit.